sys/include/net/gnrc/netif/conf.h

```diff
@@ -46,6 +46,17 @@ extern "C" {
 #define GNRC_NETIF_PRIO            (THREAD_PRIORITY_MAIN - 5)
 #endif
 
+/**
+ * @brief   Message queue size for network interface threads
+ *
+ * @attention   This has influence on the used stack memory of the thread, so
+ *              the thread's stack size might need to be adapted if this is
+ *              changed.
+ */
+#ifndef GNRC_NETIF_MSG_QUEUE_SIZE
+#define GNRC_NETIF_MSG_QUEUE_SIZE  (16U)
```

Comment:
Previous setting was 8. Would make sense to keep that value.

Reply:
ah, nvm, PR text describes the change. ok as is.

Reply:
See OP:

> 1. Increases the queue size from 8 to 16 for `gnrc_netif` threads and

Also I put the change of the message queue size in a separate commit + added testing procedure description to account for the change ;-).